packages/office-ui-fabric-react/src/components/Shimmer/ShimmerElementsGroup/ShimmerElementsGroup.styles.ts

```diff
@@ -0,0 +1,24 @@
+import { IShimmerElementsGroupStyleProps, IShimmerElementsGroupStyles } from './ShimmerElementsGroup.types';
+import { getGlobalClassNames } from '../../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-ShimmerElementsGroup-root'
+};
+
+export function getStyles(props: IShimmerElementsGroupStyleProps): IShimmerElementsGroupStyles {
+  const { width, flexWrap, theme } = props;
+
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  return {
+    root: [
+      classNames.root,
+      {
+        display: 'flex',
+        alignItems: 'center',
+        flexWrap: flexWrap ? 'wrap' : 'nowrap',
+        width: width ? width : 'auto'
```

Comment:
For any values that could be very dynamic, please use inline styling. `width` could be 300px or 20px or perhaps animated. Each unique width would generate a new classname. We could certainly avoid that by just inlining and not merging into a classname.

Reply:
Fixed.